tests/jms-tests/src/test/java/org/apache/activemq/artemis/jms/tests/MessageConsumerTest.java

```diff
@@ -2285,7 +2286,6 @@ public void run() {
       }
    }
 
-   // This is commented out until http:// jira.jboss.com/jira/browse/JBMESSAGING-983 is complete
    @Test
    public void testStopConnectionDuringOnMessage() throws Exception {
```

Comment:
This is a very old test... probably there's another similar test on our testsuite already.

Perhaps you could check if there is another test doing this already and close this?

Reply:
I mean.. delete this.